src/services/keycloakConfig.ts

```diff
@@ -0,0 +1,6 @@
+export const keycloakConfig = {
```

Comment:
For a moment this stays hardcoded, but would be nice to extract is to some form of configuration